front/public/locales/fr/operationalStudies/project.json

```diff
@@ -71,6 +69,8 @@
   "studiesCount_one": "1 étude",
   "studiesCount_other": "{{count}} études",
   "studiesCount_zero": "Aucune étude",
+  "studyBusinessCode": "Code business",
```

Comment:
:sadface:

Reply:
Could you be more specific? :eyes:

Reply:
🇫🇷 https://www.academie-francaise.fr/business 🇫🇷

Reply:
anyway, we can replace with "exploitation"

Reply:
I don't think it's appropriate to use informal wording here. Also the meaning of this definition doesn't quite fit the bill IMHO.

Reply:
Hahaha ok, done